engine room additions & minor changes

MuckerMayhem opened the pull request:
:cl: Mucker, SilentAutumn
maptweak: Added a direct fuel line from the H2 store to the cold/hot loops.
maptweak: Moved the engine room APC into the main airlock.
maptweak: Changed the pumps from hot/cold loop to wasteline to regulators.
maptweak: Named all of the various pumps and regulators in the engine room.
maptweak: Removed the CO2 cans from the engine room and 2 of the H2 cans (no longer need 4 of them).
/:cl:

No more canister hauling for hydrogen-based setups (hooray!).
The APC was moved as it was in an awkward spot and tended to be toasted any time something exploded, so this makes managing emergencies involving the engine a little nicer in its new position.

HeyBanditoz reviewed (dismissed):
While you're here you should delete this accidental `/obj` here in engi
<img width="1371" height="818" alt="strongdmm-v2 16 0 alpha-x86_64-pc-windows-gnu exe old_FN4urOPV4p" src="https://github.com/user-attachments/assets/6bce5acc-6ba1-4d2e-b629-8e52bcb23cb8" />

MuckerMayhem committed:
engine room improvements

Added a direct H2 line to the cold/hot loops with regulators to set pressure

changed a couple pumps to regulators

moved the engine room APC into the engine room airlock for easier access

moved the locker in the engine room airlock to the monitoring room

Removed the CO2 cans and 2 H2 cans (CO2 was never used, no longer need 2 H2 cans).

SierraKomodo reviewed (approved):
Not seeing any other issues or concerns on my end.

I am curious to see how the direct line to H2 goes. That's a REAL easy way to absolutely flood the pipes.

Spookerton reviewed (approved):
We've had some back and forth in dev on this and I think the line is going to be pretty ungood, but we can certainly see how it plays out
